### ISSUE TITLE
Error when user w/o cert tries to update a site

### ISSUE THREAD

**Issue by splix** [HelloZeroNet/ZeroNet#249]
Right now if a site [by mistake] tries to call sign command from a user w/o any certificate (like a user that never registered in zeroid.bit) then server throws an internal exception. It will be helpful to show some meaningful error message instead.

There: https://github.com/HelloZeroNet/ZeroNet/blob/master/src/Content/ContentManager.py#L259
```
user_urn = "%s/%s" % (content["cert_auth_type"], content["cert_user_id"])  # web/nofish@zeroid.bit
```

It even have some workaround above this line, but it's commented out. I guess there was a reason

**Comment by HelloZeroNet** [comment]
Yeah, there is no error message callback yet, currently you can check if the user has cert or not by:
```coffeescript
@cmd "siteInfo", {}, (site_info) =>
     if site_info.cert_user_id then alert(site_info.cert_user_id)
```

**Comment by splix** [comment]
yes, I know, I do that now. 
But before I figured out, I spent few hours to understand why it's happening and what should I do to avoid such error. I mean detailed error will be really helpful for novice zeronet developers.

**Comment by grez911** [comment]
@splix What file are you trying to sign? To sign `"data/users/" + this.site_info.auth_address + "/content.json"` you need `auth_address` already but `site_info` is undefined if certificate is not selected. So I draw a conclusion that you are trying to sign something else, and that means you don't understand what are you doing.
There is no way to prevent thousands of ways to shoot yourself in the foot, and this is not an educational project.
@HelloZeroNet Please, close.

**Comment by purplesyringa** [comment]
@HelloZeroNet AFAIK this is not a problem anymore, can this issue be closed?
